Model/src/main/java/org/apidb/apicommon/model/report/ai/expression/AiExpressionCache.java

```diff
@@ -387,6 +387,11 @@ private List<JSONObject> populateExperiments(List<ExperimentInputs> experimentDa
           }))
 
         ), exec));
+
```

Comment:
I don't think this is a good idea.  We are essentially artificially slowing down the entire process, at least within a request.  For a 40-experiment gene, this line will guarantee that 20 seconds pass before we see a result, even if the entire thing is cached!

Reply:
Hmm, ah, yes, duh!